api/sql/schema/750$transfer.push.confirmIssuer.sql

```diff
@@ -13,4 +16,18 @@ WHERE
     transferId = @transferId AND
     issuerTxState = 1
 
-IF @@ROWCOUNT <> 1 RAISERROR('transfer.confirmIssuer', 16, 1);
+DECLARE @COUNT int = @@ROWCOUNT
+IF @COUNT <> 1 RAISERROR('transfer.confirmIssuer', 16, 1);
```

Comment:
raise should be last statement, as we do not want to lose the event
see pattern in other similar procedures